### ISSUE TITLE
rtorrentToSickBeard.py never fires after download

### ISSUE THREAD

**Issue by tomlarose** [xombiemp/ultimate-torrent-setup#8]
I just recently followed your guide on my ubuntu 14.04 LTS server and everything works great except  rtorrentToSickBeard.py is never ran when the download completes (or any script for that matter)

Here's the line from my rtorrent.rc
```shell
method.set_key = event.download.finished,rtorrentToSickBeard,"execute2={/home/plex/.sickbeard/autoProcessTV/rtorrentToSickBeard.py,$d.get_base_path=,$d.get_name=,$d.get_custom1=}"
```

Here are my permissions on rtorrentToSickBeard.py
`
-rwxrwxrwt 1 plex plex  419 Mar  8 22:44 rtorrentToSickBeard.py
`

I've tried replacing rtorrentToSickBeard.py to a generic script and I cannot get the event to fire at all when a download is complete.

Any help you can provide would be appreciated.

**Comment by xombiemp** [comment]
Are you using the latest GIT version of rtorrent?

Enable the execute logging with:
log.execute=/tmp/exec.log

Then download a torrent and let it finish so it tries to run the finished script.  Then post the contents of the log.

**Comment by dvizzacc** [comment]
I've got the same issue.

@xombiemp The exec.log doesn't log anything about the script execution. It seems that the event.download.finished doesn't fire at all.

@tomlarose Did you manage to get it to work?

**Comment by xombiemp** [comment]
This was fixed in 8e48a6582fb855b2c273b90c2d079d9c09e7e598.
